eth-contracts/test/delegateManager.test.js

```diff
@@ -994,10 +994,11 @@ contract('DelegateManager', async (accounts) => {
           singleDelegateAmount,
           { from: delegator })
 
-        await delegateManager.delegateStake(
+        let txReceipt = await delegateManager.delegateStake(
           stakerAccount,
           singleDelegateAmount,
-          { from: delegator })
+          { from: delegator, gas: 8000000 })
```

Comment:
why this number in particular? (maybe worth a comment)

Reply:
block gas limit